cmd/aws-actuator/main.go

```diff
@@ -355,6 +355,10 @@ func bootstrapCommand() *cobra.Command {
 				glog.Info(msg)
 			}
 
+			clusterFramework.MachineControllerImage = "openshift/origin-aws-machine-controllers:v4.0.0"
+			clusterFramework.MachineManagerImage = "openshift/origin-aws-machine-controllers:v4.0.0"
+			clusterFramework.NodelinkControllerImage = "registry.svc.ci.openshift.org/openshift/origin-v4.0-2019-01-03-031244@sha256:152c0a4ea7cda1731e45af87e33909421dcde7a8fcf4e973cd098a8bae892c50"
```

Comment:
this worth a comment for clarifying

Reply:
We need to change it to something better looking, it's only a temporary solution. Once `docker.io/openshift/origin-machine-api-operator` is updated, I will switch it to it.

Reply:
Yeah - should be very explicit as to why we are committing to said version.

Reply:
I have it ready, just waiting for Andrew's review so I can accommodate all requested changes.